concepts/index.md

```diff
@@ -6,29 +6,38 @@ layout: documentation
+openHAB is built around two distinctly different views.
+The first is the physical view - devices, configurations, connections, etc.
+The second is the function view.
+This functional view consists of how information about the physical view is presented in User Interfaces, how automation operates on representations of physical devices, or how an action on a User Interface affects the software associated with the physical device it represents.
+It can be helpful to know up front that these two views exist as you are learning about openHAB.
 
 ## Things, Channels, Items and Links
 
-**Things** are the entities that can be physically added to a system and which can potentially provide many functionalities at once. It is important to note that things do not have to be devices, but they can also represent a web service or any other manageable source of information and functionality.
-Things provide their functionality through a set of **Channels**. Channels are "passive" and can be regarded as a declaration of a Thing, what it can offer. It is up to the individual setup, which of the Channels are actively used through Items (see below).
+**Things** are the entities that can be physically added to a system.
+These Things may provide more than just one function (for example, a thermostat may allow you to set the room temperature and also turn on a fan).
+It is important to note that Things do not have to be physical devices; they can also represent a web service or any other manageable source of information and functionality.
 
-**Items** represent (fine-grained) functionality that is used by applications - as user interfaces or automation logic. Items have a state and they can receive commands.
+Things provide their functionality through a set of **Channels**.
+Channels are "passive" and can be regarded as a declaration of a Thing; what it can offer.
+Whether an openHAB installation takes advantage of a particular capability offered through a Thing's Channel depends on whether it has been configured to do so.
+openHAB makes use of Channels through links you establish to Items when you configure your system (see below).
 
-The glue between Things and Items are **Links**. Links are associations between exactly one Thing Channel and one Item.
-If a Channel is linked to an Item, it is "enabled", which means that the functionality that the Item represents is handled through the given Channel.
+**Items** represent (fine-grained) functionality that can be used by applications, either in user interfaces or automation logic.
+Items have a state and they can receive commands.
+
+The glue between Things and Items are **Links**.
+Links are associations between exactly one Thing Channel and one Item.
+If a Channel is linked to an Item, it is "enabled", which means that the functionality that the Item represents is accessible through the Channel.
```

Comment:
Maybe its my non native english again, but i think there is one `that` more than needed.

Reply:
Agree.  Just so you know, it is acceptable gramatically as written, but I agree that taking the extra `that` out improves readability.  This will be addressed in my next commit.